### DISCUSSION TITLE
Using platform-native control styles when targeting WinUI 3 do not work through legacy namespace

### DISCUSSION THREAD

**Discussion by marwalsch** [unoplatform/uno#7729] [Q&A] [answered]
## On which page?
https://platform.uno/docs/articles/native-styles.html

## What's wrong?
When using the WinUi 3 project template, the package including the `Uno.UI` namepsace is no longer included by default. Adding the `Uno.UI` package to  the projects for the sake of enabling `FeatureConfiguration` however causes several types to become ambigious amongst the Microsoft based namespaces used by WinUi (the project will also not built, even if resolving those ambiguities);

`ApplicationExecutionState` is included Microsoft.Windows.SDK.NET as well, so is
`SuspendingEventArgs`.

(inside the `App.xaml.cs`).

The page does not acknowledge the existence of an alternative namepspace (or is there none at all?).

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. The Uno.UI namespace is available when using WinUI 3 compatible namespace (Uno.WinUI). Note that this namespace is not available in Windows desktop projects, and you'll need to surround it using `#if HAS_UNO` so that the Windows tooling ignores the statement.
